core/src/main/groovy/noe/ews/server/tomcat/TomcatProperties.groovy

```diff
@@ -11,13 +11,21 @@ import noe.common.utils.Library
  *
  */
 class TomcatProperties {
+  public static final String EWS_VERSION
   public static final String TOMCAT_MAJOR_VERSION
   public static final String PUBLIC_IP_ADDRESS
 
   static {
+
+    EWS_VERSION = Library.getUniversalProperty('ews.version')
```

Comment:
There is `noe.common.DefaultProperties#ewsVersion`, could we please use it instead of string property.
Because we have way to get `ews.version`, I woudn't create new public field in this class to prevent possible confusion in a future.

Yes, there is lot of stuff in noe-core to get familar with :).

Reply:
OK, will do
There is a lot, but that's half of the fun :)